docs/usage/integration.md

```diff
@@ -0,0 +1,413 @@
+# Integration
+
+TigerBeetle is designed to guard against bugs not only in its [own code](../TIGER_STYLE.md), but
+at the boundaries, in the application code which interfaces with TigerBeetle.
+This is exhibited by the client's API design, which may be [surprising](#retries) when contrasted
+with a more conventional database.
+
+Strict consistency guarantees at the database level simplify application logic and error handling
+farther up the stack.
+
+
+
+## Consistency
+
+TigerBeetle provides strict serializability (serializability + linearizability) to each
+[client session](#client-sessions).
+
+But consistency models are notoriously arcane — what specifically does this mean for applications
+in practice?
+
+- [**Client sessions**](#client-sessions)
+  - A client session may have at most one in-flight request.
+  - A client session reads its own writes.
+  - A client session observes writes in the order that they occur on the cluster.
+  - A client session observes [`debits_posted`](../reference/accounts.md#debits_posted) and
+    [`credits_posted`](../reference/accounts.md#credits_posted) as monotonically increasing.
```

Comment:
Maybe one more followup sentence to clarify further? `as monotonically increasing. Within a client session, you will never see credits_posted or debits_posted decrease.` Though you are trying to avoid `you`...

Reply:
:+1: Good clarification, "monotonically increasing" is still a little obtuse. How about

> That is, a client session will never see `credits_posted` or `debits_posted` decrease.

Reply:
Yep that's good